components/control_bar.py

```diff
@@ -711,6 +712,23 @@ def create_reset_view_affix():
     )
 
 
+def create_viewfinder_affix():
+    return dmc.Affix(
+        position={"top": 0, "right": 0},
+        zIndex=9999999,
+        children=dmc.Tooltip(
```

Comment:
Can you use the built in `components.control_bar._tooltip()` component so that the styling matches the rest of the app?